psi4/driver/qcdb/util/misc.py

```diff
@@ -0,0 +1,110 @@
+import re
+import sys
+import math
+
+import numpy as np
+
+from ..physconst import psi_bohr2angstroms
+
+if sys.version_info >= (3, 0):
+    basestring = str
+
+
+def distance_matrix(a, b):
+    """Euclidean distance matrix between rows of arrays `a` and `b`. Equivalent to
+    `scipy.spatial.distance.cdist(a, b, 'euclidean')`. Returns a.shape[0] x b.shape[0] array.
+
+    """
+    assert a.shape[1] == b.shape[1], """Inner dimensions do not match"""
+    distm = np.zeros([a.shape[0], b.shape[0]])
+    for i in range(a.shape[0]):
```

Comment:
Maybe `distm[i] = np.linalg.norm(a[i] - b, axis=1)` to get a little broadcasting in?